sequence_processing_pipeline/Commands.py

```diff
@@ -0,0 +1,151 @@
+import glob
+import gzip
+import os
+from os.path import split, join, isdir
+from os import makedirs
+
+
+def fake_directory_structure_for_testing():
+    with open('X.decimated', 'r') as f:
+        count = 0
+        for line in f:
+            count += 1
+            line = line.strip()
+            print(count)
+            spath, sfile = split(line)
+            makedirs(spath, exist_ok=True)
+            if not isdir(line):
+                with open(line, 'w') as f2:
+                    f2.write("Hello")
+            else:
+                print(".")
+
+
+def split_similar_size_bins(data_location_path, max_file_list_size_in_gb,
+                            batch_prefix):
+    '''
+
+    :param data_location_path: Path to the ConvertJob directory.
+    :param max_file_list_size_in_gb: Upper threshold for file-size.
+    :param batch_prefix: Path + file-name prefix for output-files.
+    :return: The number of output-files created.
+    '''
+
+    # SPP root paths are of the form:
+    # ../72dc6080-c453-418e-8a47-1ccd6d646a30/ConvertJob, and contain only
+    # directories named after projects e.g:
+    # 72dc6080-c453-418e-8a47-1ccd6d646a30/ConvertJob/Tell_Seq_15196.
+    # Each of these directories contain R1 and R2 fastq files. Hence, path
+    # is now the following:
+    fastq_paths = glob.glob(data_location_path + '*/*.fastq.gz')
```

Comment:
This assumes the fastq.gzs fall under a <qiita_study_id>/<qiita_artifact_id> structure, which I dont think is assured here

Reply:
This should be okay, actually. I ran the code as it was against your original data directory and reviewed the output. I then adjusted it so that it would work with where we expect the raw fastq files to be in SPP. Basically we expect our fastq files to just be separated by project, instead of project and then prep.